bin/setup

```diff
@@ -11,7 +11,7 @@ def setup
 
   log "Installing migrations from dependencies"
   # Copies migrations that don't already exist into the `db/migrate` directory
-  system! "bin/rails generate catalogue:patrons:install"
```

Comment:
Just being paranoid - you meant to remove the patrons db stuff?

Reply:
It's pointing at the wrong install generator. The script name changed when I moved it to `nla-blacklight_common`.